projects/robots/mir/mir100/protos/Mir100.proto

```diff
@@ -0,0 +1,668 @@
+  field  MFNode      frontLidarSlot   []
+  field  MFNode      backLidarSlot    []
+  field  MFNode      bodySlot         []
+]
+{
+  Robot {
+    translation IS translation
+    rotation IS rotation
+    controller IS controller
+    controllerArgs IS controllerArgs
+    customData IS customData
+    supervisor IS supervisor
+    synchronization IS synchronization
+    name IS name
+    children [
+      Transform {
+        translation 0.037646 0 0
+        children [
+          Shape {
+            appearance DEF WHITE Plastic {
+            }
+            geometry Mesh {
+              url [
+                "meshes/mir_100_base.dae"
+              ]
+            }
+          }
+          Shape {
+            appearance DEF BLACK Plastic {
+              colorOverride 0 0 0
+            }
+            geometry Mesh {
+              url [
+                "meshes/mir_100_base_corners.dae"
+              ]
+            }
+          }
+        ]
+      }
+      DEF IMU_LINK Solid {
+        translation 0 0 0.25
+        children [
+          Accelerometer {
+            name "imu_data accelerometer"
+            lookupTable [
+              -100 -100 -0.002
+              100 100 0.002
+            ]
+          }
+          Gyro {
+            name "imu_data gyro"
+            lookupTable [
+              -100 -100 -0.002
+              100 100 0.002
+            ]
+          }
+          Compass {
+            name "imu_data compass"
+            lookupTable [
+              -1 -1 -0.2
+              1 1 0.2
+            ]
+          }
+          DEF IMU_FRAME Solid {
+            name "imu_frame"
+          }
+        ]
+        name "imu_link"
+      }
+      Transform {
+        translation 0.4288 0.2358 0.1954
+        rotation -0.2811 -0.6786 0.6786 2.5936
+        children IS frontLidarSlot
+      }
+      Transform {
+        translation -0.3548 -0.2352 0.1954
+        rotation 0.8629 -0.3574 0.3574 -1.7178
+        children IS backLidarSlot
+      }
+      Transform {
+        translation 0.037646 0 0.349
+        children IS bodySlot
+      }
+      Transform {
+        translation 0.03733 0.0005 0.2085
+        children [
+          Shape {
+            appearance PBRAppearance {
+              baseColor 0 0.9 1
+              emissiveColor 0 0.9 1
+              transparency 0.5
+            }
+            geometry DEF SIDE_LIGHT Box {
+              size 0.785 0.47 0.053
+            }
+          }
+        ]
+      }
+
+      DEF US_1_FRAME Solid {
+        translation 0.45 -0.12 0.16
+        name "us_1_frame"
+      }
+      DEF US_2_FRAME Solid {
+        translation 0.45 0.12 0.16
+        name "us_2_frame"
+      }
+      DEF LEFT_WHEEL_JOINT HingeJoint {
+        jointParameters HingeJointParameters {
+          axis 0 1 0
+          anchor 0 0.222604 0.0625
+        }
+        device [
+          RotationalMotor {
+            name "mir100_middle_left_wheel_joint"
```

Comment:
Other robots don't have their name as prefix in names. You should just name them `middle_left_wheel_joint`, `middle_left_wheel_joint_sensor`, ...